.changeset/sweet-suns-breathe.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
while we're touching this package could we remove the `console.log` that's in `packages/code/src/renderingPlugin.spec.tsx`

Reply:
Nice catch, done.